### ISSUE TITLE
Thousands separator

### ISSUE THREAD

**Issue by alcarazr** [matze/mtheme#173]
By looking at the 'pgfplotsthemetol.dtx' file, the default thousands separator (a comma) should be disabled, according to the code (which applies to both mlineplot and mbarplot):
```TeX
disable thousands separator/.style={
    /pgf/number format/.cd,
      1000 sep={}
  }
```

However I am seeing the comma thousands separator. A MWE, consisting of just one slide with a bar chart, would be:

```TeX
\documentclass{beamer}
\usetheme{metropolis}           % Use metropolis theme
\usepackage{pgfplots}

\begin{document}
  \begin{frame}{First Frame}
\begin{figure}
    \begin{tikzpicture}
      \begin{axis}[
        mbarplot,
        xlabel={Foo},
        ylabel={Bar},
        width=0.9\textwidth,
        height=6cm,
      ]

      \addplot plot coordinates {(1000, 20) (2000, 25) (3000, 22.4) (4000, 12.4)};
      \addplot plot coordinates {(1000, 18) (2000, 24) (3000, 23.5) (4000, 13.2)};
      \addplot plot coordinates {(1000, 10) (2000, 19) (3000, 25) (4000, 15.2)};

      \legend{lorem, ipsum, dolor}

      \end{axis}
    \end{tikzpicture}
  \end{figure}
  \end{frame}
\end{document}
```

Is this expected?

Thank you very much,

Rodrigo

**Comment by alcarazr** [comment]
I forgot to attach a preview (pdf) of what I get after compiling.
[MWE.pdf](https://github.com/matze/mtheme/files/120307/MWE.pdf)

**Comment by benjamin-weiss** [comment]
No, that is not what the code does. There is an additional style defined named `disable thousands separator`. In the default settings this style is not used. You can add it to your plots like this:

```TeX
\documentclass{beamer}
\usetheme{metropolis} % Use metropolis theme
\usepackage{pgfplots}

\begin{document}
\begin{frame}{First Frame}
\begin{figure}
\begin{tikzpicture}
\begin{axis}[
mbarplot,
xlabel={Foo},
ylabel={Bar},
width=0.9\textwidth,
height=6cm,
disable thousands separator
]

  \addplot plot coordinates {(1000, 20) (2000, 25) (3000, 22.4) (4000, 12.4)};
  \addplot plot coordinates {(1000, 18) (2000, 24) (3000, 23.5) (4000, 13.2)};
  \addplot plot coordinates {(1000, 10) (2000, 19) (3000, 25) (4000, 15.2)};

  \legend{lorem, ipsum, dolor}

  \end{axis}
\end{tikzpicture}
\end{figure}
\end{frame}
\end{document}
```

PS: You can add code with multiple lines like this:

    ```TeX

    my tex code

    ```

**Comment by alcarazr** [comment]
I see. Thank you very much and please accept my apologies for asking such simple questions. I really appreciate your time.

PS: Thanks for the tip about multiple lines of code ;)

**Comment by benjamin-weiss** [comment]
No problem. I'm happy to help. You can also edit your original post to test multiline code. ;)

Please close your issues once they are resolved to keep the issue tracker tidy.

**Comment by alcarazr** [comment]
I am reopening this issue just to suggest to include this feature in the manual. Right now, unless one takes a look at the source code, there is no way to know its existence 

Thanks!

**Comment by matze** [comment]
This is fixed with 6741e329efcaa, thanks for the suggestion!
